lib/widgets/speaker_detail/speaker_detail_information.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:hackathon_flutterturkiye_org/core/styles/theme_helper.dart';
+import 'package:hackathon_flutterturkiye_org/core/utils/responsive_helper.dart';
+import 'package:hackathon_flutterturkiye_org/widgets/speaker_detail/speaker_detail.dart';
+
+class SpeakerInformation extends StatelessWidget {
+  const SpeakerInformation({
+    Key key,
+    @required this.speakerName,
+    this.speakerCompany,
+    this.speakerTwitter,
+    this.speakerGithub,
+    this.speakerLinkedIn,
+  }) : super(key: key);
+
+  final String speakerName;
+  final String speakerCompany;
+  final String speakerTwitter;
+  final String speakerGithub;
+  final String speakerLinkedIn;
+
+  @override
+  Widget build(BuildContext context) {
+    final screenSize = MediaQuery.of(context).size;
+
+    var crossAxisAlignment = CrossAxisAlignment.start;
+    var textAlign = TextAlign.start;
+    var dividerWidth = screenSize.width * 0.6;
+    var speakerNameFontSize = 88.0;
+
+    if (ResponsiveHelper.isMediumScreen(context)) {
+      speakerNameFontSize = 72.0;
+    } else if (ResponsiveHelper.isSmallScreen(context)) {
+      crossAxisAlignment = CrossAxisAlignment.center;
+      textAlign = TextAlign.center;
+      dividerWidth = screenSize.width * 0.8;
+      speakerNameFontSize = 60.0;
+    }
+
+    return Column(
+      crossAxisAlignment: crossAxisAlignment,
+      mainAxisAlignment: MainAxisAlignment.end,
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        SpeakerName(
+          speakerName: speakerName,
+          textAlign: textAlign,
+          speakerNameFontSize: speakerNameFontSize,
+        ),
+        // Divider
+        SpeakerDetailDivider(width: dividerWidth),
+        // Space
+        const SizedBox(height: 16.0),
+        // Company
+        SpeakerCompany(speakerCompany: speakerCompany, textAlign: textAlign),
+        // Space
+        const SizedBox(height: 24.0),
+        // Social Medias
+        SpeakerSocialMedias(
+          twitter: speakerTwitter,
+          github: speakerGithub,
+          linkedin: speakerLinkedIn,
+        ),
+      ],
+    );
+  }
+}
+
+class SpeakerName extends StatelessWidget {
+  const SpeakerName({
+    Key key,
+    @required this.speakerName,
+    @required this.textAlign,
+    @required this.speakerNameFontSize,
+  }) : super(key: key);
+
+  final String speakerName;
+  final TextAlign textAlign;
+  final double speakerNameFontSize;
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      speakerName,
+      textAlign: textAlign,
+      style: TextStyle(
```

Comment:
Can this style be constant? Or even text.

Reply:
Because of the **speakerName**, Text cannot be a constant. TextStyle also cannot because of the **fontSize**.

Reply:
> Can this style be constant? Or even text.

![Screen Shot 2021-02-09 at 17 07 34](https://user-images.githubusercontent.com/26827403/107375003-50353f80-6af9-11eb-8c22-57ae9d269cb7.png)

![Screen Shot 2021-02-09 at 17 07 59](https://user-images.githubusercontent.com/26827403/107375044-5deac500-6af9-11eb-9f12-331f678d8d64.png)